docs/guide/warnings.rst

```diff
@@ -846,18 +846,18 @@ List Of Warnings
    simulate correctly.
 
 
-.. option:: LITENDIAN
+.. option:: ASCENDINGRANGE
```

Comment:
The old LITENDIAN should remain and be documented as renamed, then add the new warning in the appropriately sorted point.